actionpack/lib/action_controller/metal/exceptions.rb

```diff
@@ -3,14 +3,9 @@ class ActionControllerError < StandardError #:nodoc:
   end
 
   class BadRequest < ActionControllerError #:nodoc:
-    attr_reader :original_exception
-
-    def initialize(type = nil, e = nil)
```

Comment:
We'll need to deprecate apps instantiating `BadRequest.new(:foo, exception)` as well, so we may need to keep the old method signature until 5.1.